devbox/ssh.sh

```diff
@@ -21,8 +21,12 @@ source "${GRAPL_DEVBOX_CONFIG}"
 ########################################
 
 declare -a PORT_FORWARDING_ARGS=()
-if [ -v FORWARD_PORT ]; then
-    PORT_FORWARDING_ARGS+=("-L" "${FORWARD_PORT}:localhost:${FORWARD_PORT}")
+if [ -v FORWARD_PORTS ]; then
+    # FORWARD_PORTS_ARRAY = FORWARD_PORTS.split(",")
```

Comment:
Do you want to delete this line?

Reply:
no, it's a "hey for those that don't know what IFS is" comment